type.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Kind is an enumeration type representing the physical types supported by the
 // parquet type system.
-type Kind int16
+type Kind int8
```

Comment:
Just want to confirm you wanted to make this change, since it's not mentioned in the PR description and goes the opposite direction of the change described in the PR description

Reply:
Yes, I forgot to mention it, just adjusted this type to match the size we use in `parquet.Value.kind`.

It's true this could be made `int`, it's a bit ambiguous since it's a type definition so applications have to do a conversion anyways if they use the value in contexts where it comes from a different integer type... 🤷  not sure what the _right_ thing to do is.